<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-MSUCRPV" name="test2" tests="4" failures="1" timestamp="23 Jun 2019 07:55:03 GMT" time="25.443" errors="0">
  <testcase name="test1" time="7.02" classname="pages.DemoTestng"/>
  <testcase name="test2" time="4.742" classname="pages.DemoTestng"/>
  <testcase name="test3" time="9.318" classname="pages.DemoTestng"/>
  <testcase name="test4" time="0.003" classname="pages.DemoTestng">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 2 parameters but the method pages.DemoTestng#test4 takes 3 and TestNG is unable in inject a suitable object">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 2 parameters but the method pages.DemoTestng#test4 takes 3 and TestNG is unable in inject a suitable object
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	... 13 more
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
</testsuite> <!-- test2 -->
